content/en/docs/languages/php/instrumentation.md

```diff
@@ -24,29 +135,23 @@ dependencies for `psr/http-client-implementation` and
 composer require guzzlehttp/guzzle
 ```
 
-Now you can install the OpenTelemetry SDK, and OTLP exporter:
+Now you can install the OpenTelemetry API and
```

Comment:
this is definitely still installing the SDK (API was installed a few lines up)

Reply:
```suggestion
Now you can install the OpenTelemetry SDK, and OTLP exporter:
```

I will revert that change, not sure why I had it in the first place